.github/workflows/export-dynamic.yaml

```diff
@@ -54,19 +49,19 @@ jobs:
     outputs:
       node-version: ${{ steps.set-env-vars.outputs.NODE_VERSION }}
       janus-cli-version: ${{ steps.set-env-vars.outputs.JANUS_CLI_VERSION }}
+      plugins-repo: ${{ steps.set-env-vars.outputs.PLUGINS_REPO }}
       workspaces: ${{ steps.gather-workspaces.outputs.workspaces }}
-      upload-project-on-error: ${{ steps.set-env-vars.outputs.UPLOAD_PROJECT_ON_ERROR }}
       overlay-repo-ref: ${{ steps.set-overlay-repo-ref.outputs.OVERLAY_REPO_REF }}
 
     steps:
-      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - uses: actions/checkout@v4.2.2
         if: ${{ github.event.inputs.overlay-branch == '' }} 
```

Comment:
Afair, the way to retrieve the inputs of a `workflow-call` workflow, is the following syntax:

`inputs.overlay-branch` (no `github.event.` prefix).

This stands for the whole file I guess.